### DISCUSSION TITLE
After 08/19/2024 update...

### DISCUSSION THREAD

**Discussion by rottenhauer** [cafali/SnapKey#66] [Q&A] [answered]
Is the software still working? Has anyone tested it to check?

**Comment by minteeaa** [comment, marked as answer]
no, this and any other Snap Tap or SOCD similar keyboard feature is officially banned for usage in official CS2 matches; the game will kick you upon detection of this feature

**Comment by cafali** [comment]
Mr. Gaben disallowed everything
